docs/pages/server-access/getting-started.mdx

```diff
@@ -12,7 +12,7 @@ This guide introduces some of these common scenarios and how to interact with Te
 2. Introspect the cluster using Teleport features.
 
 <Admonition type="tip" title="Tip">
-  This guide also demonstrates how to configure Teleport Nodes using the **bastion pattern** so that only a single Node can be accessed publicly.
+  This guide also demonstrates how to configure Teleport SSH Service using the **bastion pattern** so that only a single Node can be accessed publicly.
```

Comment:
```suggestion
  This guide also demonstrates how to configure the Teleport SSH Service using the **bastion pattern** so that only a single Node can be accessed publicly.
```

Also, why have we emphasized "bastion pattern" here? Is this a commonly known term that we often use in the docs? It's the first time I've seen it.

Reply:
@zmb3 not sure the bastion item here.  It's probably less needed then we used to compare the Teleport solution to.